### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by subwaystation**
This will bring in the AsciiDocs and Manpages for ODGI.

### CODE REVIEW

**Comment by subwaystation** [comment]
@ekg Could you please take a look? Would that correspond to what you had in mind? 
Just type `man ./odgi_build.1` to see the manpage in LINUX formatted style.

File: docs/odgi_stats.adoc [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+*-V, --set-coverage*::
+  Provide a histogram of coverage ofer unique set of paths. Print two columns to stdout: *cov*, *sets*. *cov* implies #nucleotides. *sets* lists the unique set of paths in a comma separated list. Sets with a *cov* of one and no paths in *sets* are listed, too.
+
+*-M, --multi-coverage*::
+  Provide a histogram of coverage over unique multisets of paths. Print two columns to stdout: *cov*, *sets*. *cov* implies #nucleotides. *sets* lists the unique multisets of paths in a comma separated list. Multisets with a *cov* of one and no paths in *sets* are listed, too. 
+
+=== BED Interval
+
+*-B, --bed-multicov*=_BED_::
```
Comment (subwaystation) [inline docs/odgi_stats.adoc]: @ekg How would the output of such a table look like? Also tab delimited?

File: docs/odgi_stats.adoc [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+*-b, --base-content*::
+  Describe the base content of the graph. Print to stdout the #A, #C, #G and #T of the graph.
+
+*-C, --coverage*::
+  Provide a histogram of path coverage over bases in the graph. Print three columns to stdout: *type*, *cov*, *N*. *type* is one of _full_ or _uniq_ and determines if the histogram corresponds to the full graph or only to a unique paths graph. *cov* implies the #paths. *N* implies the #nucleotides.
+
+*-V, --set-coverage*::
+  Provide a histogram of coverage ofer unique set of paths. Print two columns to stdout: *cov*, *sets*. *cov* implies #nucleotides. *sets* lists the unique set of paths in a comma separated list. Sets with a *cov* of one and no paths in *sets* are listed, too.
+
+*-M, --multi-coverage*::
```
Comment (subwaystation) [inline docs/odgi_stats.adoc]: @ekg I fail to see the difference between `-V` and `-M`. Would you have an example?

Comment (ekg) [reply docs/odgi_stats.adoc]: -V provides coverage of the _set_ of paths.

-M provides coverage of the _multiset_ (combination with possible repetition of paths).

Comment (subwaystation) [reply docs/odgi_stats.adoc]: Thanks!

**Comment by subwaystation** [comment]
@ekg This is ready to get merged.
Overview of changes:

- Added documentation for `odgi` and each of its subcommands.
- Documentation is available in HTML, PDF or TROFF format.
- I ensured that, for each subcommand, mandatory arguments are checked so that we don't get unecessary output.
- New `version` subcommand: Each time `odgi` is build, the current version is inferred via `git describe --always --tags`. Assuming, [version.cpp](./src/version.cpp) is up to date, `odgi version` will not only print out the current tagged version, but its release codename, too. 
- Automated release preparation: Each time we make a new release, we invoke [prepare_release.sh](./scripts/prepare_release.sh)  (`cd` into folder [scripts](./scripts) first!) with a new release version and codename. [version.cpp](./src/version.cpp) is updated and the documentation version is bumped up.
 Thanks :)

**Comment by ekg** [comment]
Wow! Great!

**Pull request merged**